client/rest-high-level/src/main/java/org/elasticsearch/client/ml/dataframe/DataFrameAnalyticsConfig.java

```diff
@@ -178,23 +169,6 @@ public String toString() {
         private FetchSourceContext analyzedFields;
         private ByteSizeValue modelMemoryLimit;
 
-        private Builder() {}
```

Comment:
As we have `builder()` methods to the respective classes, I wonder if we should have private constructors for the builders. Doing so would mean users would have a single way to access the builders. I am not quite sure if this makes sense, but it feels like it'd force users to write more consistent code.

Reply:
Definitely a good idea! Done.